app/views/swiss/home.scala

```diff
@@ -58,8 +58,8 @@ object home {
             td(cls := "infos")(
               span(cls := "rounds")(
                 s.isStarted option frag(s.round.value, " / "),
-                s.settings.nbRounds,
-                " rounds Swiss"
+                trans.swiss.nbRounds.pluralSame(s.settings.nbRounds),
+                " Swiss"
```

Comment:
You probably want to include "swiss" here? Otherwise it won't translate well.

Reply:
And maybe it should include the whole round info as a placeholder? There might be languages where it makes more sense to move that part somewhere else, e.g. after the text. Though maybe it needs two strings, one if it's only the max rounds and one when it's `2 / 5` 🤔

Reply:
> And maybe it should include the whole round info as a placeholder
This makes sense: Initially, I thought "Swiss" should be treated separately, but now I'm realising that it might not work out in RTL languages

> Though maybe it needs two strings, one if it's only the max rounds and one when it's 2 / 5 :thinking:
Agreed, there was previously another string for "y rounds" but I removed it. I think this is the way to go.